client/packages/common/src/utils/functions/ApiUtils.tsx

```diff
@@ -4,9 +4,9 @@ export const setNullableInput = <NODE, KEY extends keyof NODE>(
   key: KEY,
   entity?: NODE | null | undefined
 ) => {
-  if (entity == null) return { value: undefined };
+  if (entity === null) return { value: null };
 
-  if (!entity) return undefined;
+  if (entity === undefined) return undefined;
```

Comment:
Not completely required for this use case, but better to be explicit here. `!entity` would also match `0` or `""` .